platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolutionIssuesIntegrationTest.groovy

```diff
@@ -81,139 +81,6 @@ class ResolutionIssuesIntegrationTest extends AbstractIntegrationSpec {
         succeeds("resolve")
     }
 
-    @Issue("https://github.com/gradle/gradle/issues/26145#issuecomment-1957776331")
-    def "partially minimized reproducer -- android project causes corrupt serialized resolution result"() {
```

Comment:
Moved to `ResolutionIssuesRegressionIntegrationTest`. This file should only contain _unresolved_ issues